addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -23,15 +23,28 @@
                         <field name="has_multiple_providers" invisible="1"/>
                         <field name="description"/>
                         <field name="amount"/>
+                        <field name="is_amount_allowed" invisible="1"/>
                         <field name="currency_id" invisible="1"/>
                         <field name="payment_provider_selection"
                             attrs="{'invisible':[('has_multiple_providers', '=', False)]}"/>
                     </group>
                 </group>
+                <div name="payment-link-warning-information" class="alert alert-warning text-center fw-bold" role="alert" attrs="{'invisible': [('is_amount_allowed', '=', True)]}">
+                    <p attrs="{'invisible': [('amount_max', '!=', 0)]}">
+                        There is nothing to be paid.
+                    </p>
+                    <p attrs="{'invisible': ['|', ('amount', '&lt;=', 0), ('amount_max', '=', 0)]}">
+                        Please set an amount lower than <field name="amount_max" readonly="1" widget="monetary"/>.
```

Comment:
Probably not correctly rendered in some languages.
Can't we use a generic field to host the error message, and show the whole diff only if the field is not empty ?